Assets/Plugins/Pixel Crushers/Dialogue System/Prefabs/TimelineConversationAsset.cs

```diff
@@ -0,0 +1,15 @@
+using System.Collections;

```

Comment:
Can we put this script in "Assets/Scripts/Dialogue/" folder
Because its our own c# script, rather than a plugin prefab.